src/ngraph/frontend/onnx_import/op/argmax.cpp

```diff
@@ -14,9 +14,8 @@
 // limitations under the License.
 //*****************************************************************************
 
-#include "ngraph/op/argmax.hpp"
-#include "core/node.hpp"
-#include "ngraph/node.hpp"
+#include "argmax.hpp"
+#include "default_opset.hpp"
```

Comment:
Since `ArgMax` is not a part of the opset1 we should use opset0 here.

Reply:
I've already created `opset0.hpp` file to enable including different opsets same way.
Let's talk about it and other possible ways.